src/barrier.cpp

```diff
@@ -50,7 +54,8 @@ void char_set_string_elt(SEXP x, R_xlen_t i, const char* value) {
 
 // [[Rcpp::register]]
 SEXP* get_string_ptr(SEXP x) {
-    return STRING_PTR(x);
+    // TODO: should we deprecate this?
```

Comment:
May need a full rev.dep run to see if it is being called?

Reply:
Maybe we leave this code as-is for this PR, and then later test their removal in a separate PR, to see what breaks?

Reply:
Sounds like a plan!